src/app/bucket/bucket_file_reader.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+from google.cloud import storage
+from logging_config import logging
+
+logger = logging.getLogger(__name__)
+
+
+class BucketFileReader:
+    def __init__(self):
+        self.storage_client = storage.Client()
+
+    def get_file_from_bucket(self, filename: str, bucket_name: str):
```

Comment:
minor - couuld you type hint the return please

Reply:
Done